<!DOCTYPE html>
<html xmlns:ng="http://angularjs.org" ng-app="ddiApp" id="ng-app" xmlns:ng="http://angularjs.org" class="js flexbox canvas canvastext webgl no-touch geolocation postmessage no-websqldatabase indexeddb hashchange history draganddrop websockets rgba hsla multiplebgs backgroundsize borderimage borderradius boxshadow textshadow opacity cssanimations csscolumns cssgradients no-cssreflections csstransforms csstransforms3d csstransitions fontface generatedcontent video audio localstorage sessionstorage webworkers applicationcache svg inlinesvg smil svgclippaths ng-scope">
<head>
    <style type="text/css">@charset "UTF-8";[ng\:cloak],[ng-cloak],[data-ng-cloak],[x-ng-cloak],.ng-cloak,.x-ng-cloak,.ng-hide:not(.ng-hide-animate){display:none !important;}ng\:form{display:block;}
    </style>
        <meta charset="utf-8">
        <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
        <meta content="width=device-width" name="viewport">
        <meta content="DDI is a integrate resource to multiple omics repositories, including Proteomics, Metabolomics and Genomics" name="description">


        <title>DDI: API</title>

        <link rel="stylesheet" href="css/bootstrap.min.css">
        <link type="text/css" href="css/ddi.css" rel="stylesheet">
        <link type="text/css" href="css/main.css" rel="stylesheet">
        <!--
        <link type="text/css" href="css/prider.css" rel="stylesheet">
        <link type="text/css" href="css/ebi-global.css" rel="stylesheet">
        <link type="text/css" href="css/ebi-visual.css" rel="stylesheet">
        -->

        <link rel="stylesheet" href="//www.ebi.ac.uk/web_guidelines/css/compliance/develop/boilerplate-style.css">
        <link rel="stylesheet" href="//www.ebi.ac.uk/web_guidelines/css/compliance/develop/ebi-global.css" type="text/css" media="screen">
        <link rel="stylesheet" href="//www.ebi.ac.uk/web_guidelines/css/compliance/develop/ebi-visual.css" type="text/css" media="screen">
        <link rel="stylesheet" href="//www.ebi.ac.uk/web_guidelines/css/compliance/develop/984-24-col-fluid.css" type="text/css" media="screen">
    
 <script src="js/vendor/jquery-1.10.1.min.js"></script>
 <script src="https://google-code-prettify.googlecode.com/svn/loader/run_prettify.js"></script>
</head>

<script>
$(function(){
  $("#footer").load("footer.html"); 
});
</script>

<body class="level2">

<div id="skip-to">
    <ul>
        <li><a href="#content">Skip to main content</a></li>
        <li><a href="#local-nav">Skip to local navigation</a></li>
        <li><a href="#global-nav">Skip to EBI global navigation menu</a></li>
        <li><a href="#global-nav-expanded">Skip to expanded EBI global navigation menu (includes all sub-sections)</a></li>
    </ul>
</div>

<div id="wrapper" class="container_24">
<header>
   <div id="global-masthead" class="masthead grid_24">
   <!--This has to be one line and no newline characters-->
     <a href="//www.ebi.ac.uk/" title="Go to the EMBL-EBI homepage"><img src="//www.ebi.ac.uk/web_guidelines/images/logos/EMBL-EBI/EMBL_EBI_Logo_white.png" alt="EMBL European Bioinformatics Institute"></a>
     <nav>
     <ul id="global-nav">
     <!-- set active class as appropriate -->
       <li id="services" class=" first active"><a href="//www.ebi.ac.uk/services" title="Services">Services</a></li>
       <li id="research" class=""><a href="//www.ebi.ac.uk/research" title="Research">Research</a></li>
       <li id="training" class=""><a href="//www.ebi.ac.uk/training" title="Training">Training</a></li>
       <li id="industry" class=""><a href="//www.ebi.ac.uk/industry" title="Industry">Industry</a></li>
       <li id="about" class=" last"><a href="//www.ebi.ac.uk/about" title="About us">About us</a></li>
     </ul>
     </nav>
  </div>


  <div id="local-masthead" class="masthead grid_24 nomenu">
    <!-- local-title -->
    <!-- NB: for additional title style patterns, see http://frontier.ebi.ac.uk/web/style/patterns -->
      <div class="grid_12 alpha logo-title" id="local-title">
        <a href="/app" title="Back to DDI homepage"><img src="/app/static/img/logo/DDI-icon-3.png" alt="logo" width="64" height="64"></a>
        <span><h1><a href="/app" title="Back to DDI homepage">Datasets Discovery Index</a></h1></span>
      </div>
    <!-- /local-title -->

    <!-- local-search -->
    <!-- NB: if you do not have a local-search, delete the following div, and drop the class="grid_12 alpha" class from local-title above -->
      <div ng-controller="QueryCtrl" class="grid_12 omega">
         <form novalidate name="queryForm" class="local-search" ng-submit="submit_query()">
            <fieldset>
               <div class="form-group" ng-class="(query.submitted && queryForm.$invalid) ? 'has-error' : ''">
                  <div class="input-group">
                         <input type="text" style="background-color:#fff; width:99%"
                         placeholder="organism, repository, gene, tissue, accession"
                         class="form-control"
                         id="query-text"
                         name="text"
                         tabindex="1"
                         ng-model="query.text"
                         ng-minlength="2"
                         ng-maxlength="1000"
                         autocomplete="off"
                         required>
                         <div class="input-group-btn">
                             <button type="submit" class="btn btn-primary">
                                 <i class="fa fa-search"></i> Search
                             </button>
                         </div><!-- /input-group-btn -->
                  </div><!--input-group -->
                     <span class="help-block example-searches">
                         <small>
                             <i>
                                 Examples:
                                 <a href="" rel="nofollow" ng-click="meta_search('cancer')">cancer</a>,
                                 <a href="" rel="nofollow" ng-click="meta_search('TAXONOMY:&quot;9606&quot;')">Homo sapiens</a>,
                                 <a href="" rel="nofollow" ng-click="meta_search('Orbitrap')">Orbitrap</a>,
                                 <a href="" rel="nofollow" ng-click="meta_search('Q9HAU5')">Q9HAU5</a>,
                                 <a href="" rel="nofollow" ng-click="meta_search('Phospho')">Phospho</a>,
                                 <a href="" rel="nofollow" ng-click="meta_search('Hela')">Hela</a>
                             </i>
                         </small>
                     </span>
                     <label class="control-label ng-cloak has-error"
                            for="query-text" 
                            ng-show="query.submitted && (queryForm.text.$error.required || queryForm.text.$error.minlength)"
                            ng-cloak>
                     Your query is too short
                     </label>
                     <label class="control-label ng-cloak"
                            for="query-text" ng-show="query.submitted && (queryForm.text.$error.maxlength)"
                            ng-cloak>
                     Your query is too long
                     </label>
                 </div><!-- /form-group -->
             </fieldset>
            </form>
        </div>
        <!-- /local-search -->


        <!-- local-nav -->
        <nav>
            <ul class="grid_24" id="local-nav">
            <li class=" first" id="home-menu"><a title="Home" href="http://localhost:8000/app/">Home</a></li>
            <li class="" id="browse-menu"><a title="Browse" href="http://localhost:8000/app/browse.html#/search?q=*:*">Browse</a></li>
            <li class="" id="api-menu"><a title="DDI API" href="http://localhost:8000/app/api.html">API</a></li>
            <li class=" last" id="about-prider-menu"><a title="Databases" href="http://localhost:8000/app/databases.html">Databases</a></li>
            <!-- If you need to include functional (as opposed to purely navigational) links in your local menu,
            add them here, and give them a class of "functional". Remember: you'll need a class of "last" for
            whichever one will show up last...
            For example: -->
            <li class="functional last"><a data-icon="\" class="icon icon-static" href="http://www.ebi.ac.uk/support/index.php?query=pride">Feedback</a></li>
            <li class="functional"><a class="" href="">About</a></li>
            <li class="functional"><a class="" href="">Help</a></li>
            <li class="functional" id="login-menu" hidden="true" ><a title="Login to PRIDE Archive" data-icon="l" class="icon icon-functional" href="http://www.ebi.ac.uk/pride/archive/login">Login</a></li>
            <li class="functional first" id="register-menu" hidden="true"><a title="Register" data-icon="5" class="icon icon-functional" href="http://www.ebi.ac.uk/pride/archive/register">Register</a></li>
            </ul>
        </nav>

        <!-- /local-nav -->
</div>
</header>


<div class="container-main container">
            <!--[if lt IE 10]>
              <div class="alert alert-danger">
                You are using an <strong>outdated browser</strong> and as a result this website may not function as intended. Please <a href="http://whatbrowser.org/">upgrade your browser</a> to improve your experience.</p>
              </div>
            <![endif]-->

            <noscript>
              <div class="alert alert-danger">
                Javasript is currently disabled or is not supported by this browser. Please enable JavaScript for full functionality.
              </div>
            </noscript>

<div id="" class="row" >
</div>
<div id="" class="row" >
    <p>   
    </p>
</div>


<div id="content" class="row" >
<div id="" class="col-md-9" role="main">


  <h1 id="overview">
  <i class="fa fa-book"></i>API overview
  </h1>

<p>Most data in the Datatsets Discovery Index can be accessed programmatically using a <a href="http://localhost:9091/">RESTful API</a>
allowing for integration with other resources. The API implementation is based on the <a href="https://spring.io/guides/gs/rest-service/">Spring Rest Framework</a>.</p>

<h2 id="web-browsable-api">Web browsable API</h2>

<p>The DDI API is <strong>web browsable</strong>, which means that:</p>

<ul>
<li><p>The query results returned by the API are available in <a href="http://json.org/" target="_blank">JSON</a>format. This ensures that they can be viewed by human and accessed programmatically by computer.</p></li>
<li>The main <a href="http://localhost:9091" target="_blank">RESTful API</a> page provides a simple web-based user interface, which allows developers can familiarise themselves with the API and get a better sense of the DDI data before writing single line of code.</li>
<li><p>many resources are <strong>hyperlinked</strong> so that it's possible to navigate the API in the browser.</p></li>
</ul>

<p>As a result, developers can familiarise themselves with the API and get a better sense of the DDI data.</p>


<p><a href="http://localhost:9091/" class="btn btn-primary" role="button" target="_blank">Browse the API</a></p>

<h2 id="versioning">Versioning</h2>
            <p>
                To ensure that changes in the DDI API don't break the applications relying on it, the API is versioned, and the version is included in the API's URL.
            </p>
            <p>
                For example, the current API is at Version 1 and is available at http://www.ebi.ac.uk/ddi/ws/v1, and the next version will be available at http://www.ebi.ac.uk/ddi/ws/v2.
            </p>
            <p>
                The latest version of the API can be accessed at http://www.ebi.ac.uk/ddi/ws/current, but it's not recommended to use this endpoint for long-term development as the underlying data model may change.
            </p>
            <p>
                No backward-incompatible changes are made to each version after it's been made public. More specifically, it's guaranteed that within one version there will be no:
            </p>
            <ul>
                <li>changing urls</li>
                <li>deleting or renaming data fields</li>
                <li>changing data field types</li>
            </ul>
            <p>
                The following non-disruptive changes may be implemented to a public API:
            </p>
            <ul>
                <li>adding new endpoints</li>
                <li>adding new data fields</li>
                <li>adding new filtering methods</li>
            </ul>
            <p>
                An advance notice will be given before obsoleting an API version. To stay up to date, please consider signing up for the PRIDE <a href="https://twitter.com/DDIndex">Twitter</a> account.
            </p>
        

<hr>

<h1 id="v1">API documentation</h1>

<h2 id="v1-example-responses">Example responses</h2>

<p>Responses containing <strong>multiple entries</strong> have the following fields:</p>

<ul>
<li><em>count</em> is the number of entries in the matching set</li>
<li><em>datasets</em> is an array of datasets.</li>
<li><em>facets</em> is an array of facets.</li>
</ul>

<h4>Example</h4>
<pre>
<code  class="prettyprint">
http://localhost:9091/dataset/search?query=human
{
    "count": 733,
    "datasets": [
    {
      "id": "PXD000456",
      "source": "pride",
      "title": "Human glomerular extracellular matrix analysed by LC-MSMS",
      "description": "Extracellular matrix proteins were isolated from human glomeruli and analysed by LC-MSMS",
      "keywords": [
        "Human",
        "kidney",
        "glomerulus",
        "extracellular matrix"
      ],
      "organisms": [
        {
          "acc": "9606",
          "name": "Homo sapiens"
        }
      ],
      "publicationDate": "20140122"
    },
        // 19 more datasets 
    ],
  "facets": [
    {
      "id": "modification",
      "label": "Modification",
      "total": 181,
      "facetValues": [
        {
          "label": "Unknown modification",
          "value": "unknown modification",
          "count": "5"
        },
        //other facet values
     ],
    },
   //other facets
  ]
}
</code>
</pre>

<p>Responses containing just a <strong>single dataset</strong> have some extra navigation fields,and without the facets</p>

<pre>
<code  class="prettyprint">
http://localhost:9091/dataset/get?acc=PXD001848&database=PRIDE
{
  "id": "PXD001848",
  "name": "Global Analysis of Protein Folding Thermodynamics for Disease State Characterization, MCF7 vs MDAMB231",
  "description": "Protein biomarkers can be used to characterize and diagnose disease states such as cancer.  They can also serve as therapeutic targets. Current methods for protein biomarker discovery, which generally rely on the large-scale analysis of gene and/or protein expression levels, fail to detect protein biomarkers with disease-related functions and unaltered expression levels. Here we describe the large-scale use of thermodynamic measurements of protein folding and stability for disease state characterization and the discovery of protein biomarkers. Using the Stable Isotope Labeling with Amino Acids in Cell Culture and Stability of Proteins from Rates of Oxidation (SILAC-SPROX) technique, we assayed ~800 proteins for protein folding and stability changes in three different cell culture models of breast cancer including the MCF-10A, MCF-7, and MDA-MB-231 cell lines. The thermodynamic stability profiles generated here created distinct molecular markers for the three cell lines, and a significant fraction (~45%) of the differentially stabilized proteins did not have altered expression levels. Thus, the protein biomarkers reported here created novel molecular signatures of breast cancer and provided additional insight into the molecular basis of the disease. Our results establish the utility of protein folding and stability measurements for the study of disease processes.",
  "keywords": null,
  "publicationDate": "20150410",
  "publications": [
    {
      "id": "25825992",
      "publicationDate": "2015-04-09",
      "title": "Global analysis of protein folding thermodynamics for disease state characterization.",
      "pubabstract": "Current methods for the large-scale characterization of disease states generally rely on the analysis of gene and/or protein expression levels. These existing methods fail to detect proteins with disease-related functions and unaltered expression levels. Here we describe the large-scale use of thermodynamic measurements of protein folding and stability for the characterization of disease states. Using the Stable Isotope Labeling with Amino Acids in Cell Culture and Stability of Proteins from Rates of Oxidation (SILAC-SPROX) technique, we assayed ∼800 proteins for protein folding and stability changes in three different cell culture models of breast cancer including the MCF-10A, MCF-7, and MDA-MB-231 cell lines. The thermodynamic stability profiles generated here created distinct molecular markers to differentiate the three cell lines, and a significant fraction (∼45%) of the differentially stabilized proteins did not have altered expression levels. Thus, the differential thermodynamic profiling strategy reported here created novel molecular signatures of breast cancer and provided additional insight into the molecular basis of the disease. Our results establish the utility of protein folding and stability measurements for the study of disease processes, and they suggest that such measurements may be useful for biomarker discovery in disease.",
      "cycle": "testcyclehere"
    }
  ],
  "relatedDatasets": null,
  "data_protocol": "Peak lists were extracted from the raw LC-MS/MS data files and the data were searched against the 20265 human proteins in the 2014-04 release of the UniProt Knowledgebase (downloaded at ftp://ftp.uniprot.org/pub/databases/uniprot/current_releases/release-2014_04/knowledgebase/) using Maxquant 1.3.0.5.41 The following modifications were used: methyl methanethiosulfonate at cysteine as a fixed modification, SILAC labeling of lysine (13C614N2) and arginine (13C6), and variable (0-1) oxidation of methionine and deamidation of Asparagine and Glutamine (N and Q), and acetylation of the protein N-terminus. The enzyme was set as Trypsin, and up to 2 missed cleavages were permitted. The false discovery rate for peptide and protein identifications was set to 1%, and rest of the parameters were set at the default settings.  As part of the default settings the mass tolerance for precursor ions was set to 20 ppm for the first search where initial mass recalibration was completed and a 6 ppm precursor mass tolerance was used for the main search. The mass tolerance for fragment ions was 0.5 Da. We also included match between runs and re-quantification of the searched peptides. The search results were exported to Excel for further data analysis as described below.  Only the protein and peptide identifications with non-zero positive ratios (H/L >0) were used in subsequent data analysis steps.  The methionine-containing peptides were selected, and those methionine-containing peptides consistently identified in the protein samples derived from six or more denaturant-containing buffers were assayed.    For the methionine-containing peptides, a single averaged H/L ratio was calculated for each peptide sequence and each charge state at each denaturant concentration.  Similarly, for each analysis a median H/L ratio was determined for each protein using the H/L ratios measured for all the non-methionine-containing peptides identified in all the denaturant concentrations for a given protein.  These median H/L ratios were used to select hits with H/L>2 fold in the protein expression level analyses.  For hit peptide and protein selection in the thermodynamic analyses, all the H/L ratios generated for the non-methionine containing peptides from a given protein were divided by the median H/L ratio for that protein in order to generate normalized H/L ratios for each non-methionine containing peptide. These normalized H/L ratios were log2 transformed. The normalized and log2 transformed H/L ratios generated for the non-methionine-containing peptides in a given analysis were used to determine the 5th and 95th percentiles values used in subsequent analysis of methionine-containing peptides. The averaged H/L ratios calculated for each methionine-containing peptides were also normalized and log2 transformed. The methionine-containing peptides and proteins with log2 transformed H/L ratios less than the 5th percentile or greater than the 95th percentile values determined above were selected and then visually inspected to determine which peptides had altered H/L ratios at 2 or more consecutive denaturant concentrations to generate an initial list of protein hits.",
  "sample_protocol": "SILAC labeled MCF-7 and MDA-MB-231 cell lysates were prepared according to established SILAC protocols. Aliquots of each lysate were distributed into a series of denaturant-containing buffers, reacted with hydrogen peroxide under conditions that selectively oxidize exposed methionine residues, and quenched with the addition of excess methionine. The light and heavy samples generated at matching denaturant concentration were combined. Each combined protein sample was  submitted to a bottom-up, solution-phase, shotgun proteomics analysis using LC-MS/MS. Ultimately, L/H ratios were obtained for the peptides detected at each denaturant concentration, and the denaturant dependence of the L/H ratio’s was examined."
}
</code></pre>

<h2 id="v1-pagination">Pagination</h2>

<p>Responses containing multuple datasets are paginated to prevent accidental downloads of large amounts of data and to speed up the API.</p>
<p>The page size is controlled by the <code>size</code> parameter. Its default value is
<strong>20 datasets per page</strong>, and the maximum number of datasets per page is <strong>100</strong>.</p>
<p>Another parameter is <code>start</code> which indicates the numeric order(start from <code>0</code>, not <code>1</code>) of the first dataset in this page, Its default value is
<strong>0</strong>.</p>

<h4>Examples</h4>

<ul>
<li><a href="http://localhost:9091/dataset/search?query=human&start=0&size=50">http://localhost:9091/dataset/search?query=human&start=0&size=50</a></li>
<li><a href="http://localhost:9091/dataset/search?query=human&start=0&size=50">http://localhost:9091/dataset/search?query=human&start=0&size=20</a></li>
</ul>



<h2 id="v1-sort">Sort</h2>

<p>The result datasets can be sorted using the title,description, the publication date, the accession id and the relevance of the query term..</p>

<h4>Examples</h4>

<ul>
<li><a href="http://localhost:9091/dataset/search?query=human&sortfield=id">http://localhost:9091/dataset/search?query=human&sortfield=id</a></li>
<li><a href="http://localhost:9091/dataset/search?query=human&sortfield=publication_date">http://localhost:9091/dataset/search?query=human&sortfield=publication_date</a></li>
</ul>


<h2 id="v1-output-formats">Output formats</h2>

<p>The following output format is: <strong>JSON</strong></p>

<h2 id="v1-filtering">Filtering</h2>

<p>The API supports several filtering operations that complement the main DDI search functionality.</p>

<h3 id="v1-filtering-by-search-term">Filtering by search term</h3>

<p>There is 1 url parameter: <code>query</code></p>

<h4>Examples</h4>

<ul>
<li><a href="http://localhost:9091/dataset/search?query=human">http://localhost:9091/dataset/search?query=human</a></li>
<li><a href="http://localhost:9091/dataset/search?query=cancer">http://localhost:9091/dataset/search?query=cancer</a></li>
</ul>

<h3 id="v1-filtering-by-omics-type">Filtering by omics type</h3>

<p>The omics type can be specified by adding terms in the <code>query</code> url parameter with key: "omics_type"
(possible values: <em>Proteomics</em>, <em>Metabolomics</em>, <em>Proteomics</em>).</p>

<h4>Examples</h4>

<ul>
<li><a href='http://localhost:9091/dataset/search?query=human AND omics_type: "Proteomics"'>http://localhost:9091/dataset/search?query=human AND omics_type:"Proteomics"</a></li>
<li><a href='http://localhost:9091/dataset/search?query=human AND omics_type: "Metabolomics"'>http://localhost:9091/dataset/search?query=human AND omics_type:"Metabolomics"</a></li>
</ul>

<h3 id="v1-filtering-by-database">Filtering by database</h3>

<p>The database can be specified by adding terms in the <code>query</code> url parameter with key: "repository"
(possible values: <em>MassIVE</em>, <em>Metabolights</em>, <em>PeptideAtlas</em>, <em>PRIDE</em>).</p>

<h4>Examples</h4>

<ul>
<li><a href='http://localhost:9091/dataset/search?query=human AND repository:"MassIVE"'>http://localhost:9091/dataset/search?query=human AND repository:"MassIVE"</a></li>
<li><a href='http://localhost:9091/dataset/search?query=human AND repository:"Metabolights"'>http://localhost:9091/dataset/search?query=human AND repository:"Metabolights"</a></li>
<li><a href='http://localhost:9091/dataset/search?query=human AND repository:"PeptideAtlas"'>http://localhost:9091/dataset/search?query=human AND repository:"PeptideAtlas"</a></li>
<li><a href='http://localhost:9091/dataset/search?query=human AND repository:"PRIDE"'>http://localhost:9091/dataset/search?query=human AND repository:"PRIDE"</a></li>
</ul>

<h3 id="v1-filtering-by-organism">Filtering by Organism</h3>

<p>The organism can be specified by adding terms in the <code>query</code> url parameter with key: "TAXONOMY"
(possible values must be the TAXONOMY id: <em>9606</em>, <em>10090</em>...).</p>

<h4>Examples</h4>

<ul>
<li><a href='http://localhost:9091/dataset/search?query=human AND TAXONOMY:"9606"'>http://localhost:9091/dataset/search?query=human AND TAXONOMY:"9606"</a></li>
<li><a href='http://localhost:9091/dataset/search?query=human AND TAXONOMY:"10090"'>http://localhost:9091/dataset/search?query=human AND TAXONOMY:"10090"</a></li>
</ul>

<h3 id="v1-filtering-by-tissue">Filtering by Tissue</h3>

<p>The tissue can be specified by adding terms in the <code>query</code> url parameter with key: "tissue"
(possible values: <em>Liver</em>, <em>Cell culture</em>, <em>Brain</em>, <em>Lung</em>...).</p>

<h4>Examples</h4>

<ul>
<li><a href='http://localhost:9091/dataset/search?query=human AND tissue:"Brain"'>http://localhost:9091/dataset/search?query=human AND tissue:"Brain"</a></li>
<li><a href='http://localhost:9091/dataset/search?query=human AND tissue:"Cell culture"'>http://localhost:9091/dataset/search?query=human AND tissue:"Cell culture"</a></li>
</ul>

<h3 id="v1-filtering-by-disease">Filtering by Disease</h3>

<p>The disease can be specified by adding terms in the <code>query</code> url parameter with key: "disease"
(possible values: <em>Breast cancer</em>, <em>Lymphoma</em>, <em>Carcinoma</em>, <em>prostate adenocarcinoma</em>...).</p>

<h4>Examples</h4>

<ul>
<li><a href='http://localhost:9091/dataset/search?query=human AND disease:"Breast cancer"'>http://localhost:9091/dataset/search?query=human AND tissue:"Breast cancer"</a></li>
<li><a href='http://localhost:9091/dataset/search?query=human AND disease:"Lymphoma"'>http://localhost:9091/dataset/search?query=human AND tissue:"Lymphoma"</a></li>
</ul>


<h3 id="v1-filtering-by-modification">Filtering by Modification <small>(in proteomics)</small></h3>

<p>The Modifications (in proteomics) can be specified by adding terms in the <code>query</code> url parameter with key: "disease"
(possible values: <em>Deamidated residue</em>, <em>Deamidated</em>, <em>Monohydroxylated residue</em>, <em>Iodoacetamide derivatized residue</em>...).</p>

<h4>Examples</h4>

<ul>
<li><a href='http://localhost:9091/dataset/search?query=human AND modification:"iodoacetamide derivatized residue"'>http://localhost:9091/dataset/search?query=human AND modification:"iodoacetamide derivatized residue"</a></li>
<li><a href='http://localhost:9091/dataset/search?query=human AND modification:"monohydroxylated residue"'>http://localhost:9091/dataset/search?query=human AND modification:"monohydroxylated residue"</a></li>
</ul>

<h3 id="v1-filtering-by-instruments-platforms">Filtering by Instruments & Platforms</h3>

<p>The Instruments & Platforms can be specified by adding terms in the <code>query</code> url parameter with key: "instrument_platform"
(possible values: <em>QSTAR</em>, <em> LTQ Orbitrap</em>, <em>Q Exactive</em>, <em>LTQ</em>...).</p>

<h4>Examples</h4>

<ul>
<li><a href='http://localhost:9091/dataset/search?query=human AND instrument_platform:"LTQ"'>http://localhost:9091/dataset/search?query=human AND instrument_platform:"LTQ"</a></li>
<li><a href='http://localhost:9091/dataset/search?query=human AND instrument_platform:"Q Exactive"'>http://localhost:9091/dataset/search?query=human AND instrument_platform:"Q Exactive"</a></li>
</ul>


<h3 id="v1-filtering-by-publicationdate">Filtering by Publication Date</h3>

<p>The Publication Date can be specified by adding terms in the <code>query</code> url parameter with key: "publication_date"
(possible values: <em>2015</em>, <em>2014</em>, <em>2013</em>, <em>2014</em>...).</p>

<h4>Examples</h4>

<ul>
<li><a href='http://localhost:9091/dataset/search?query=human AND publication_date:"2015"'>http://localhost:9091/dataset/search?query=human AND publication_date:"2015"</a></li>
<li><a href='http://localhost:9091/dataset/search?query=human AND publication_date:"2014"'>http://localhost:9091/dataset/search?query=human AND publication_date:"2014"</a></li>
</ul>


<h3 id="v1-filtering-by-technologytype">Filtering by Technology Type</h3>

<p>The Technology Type can be specified by adding terms in the <code>query</code> url parameter with key: "technology_type"
(possible values: <em>Mass Spectrometry</em>, <em>Bottom-up proteomics</em>, <em>Gel-based experiment</em>, <em> Shotgun proteomics</em>...).</p>

<h4>Examples</h4>

<ul>
<li><a href='http://localhost:9091/dataset/search?query=human AND technology_type:"Mass Spectrometry"'>http://localhost:9091/dataset/search?query=human AND technology_type:"Mass Spectrometry"</a></li>
<li><a href='http://localhost:9091/dataset/search?query=human AND technology_type:"Shotgun proteomics"'>http://localhost:9091/dataset/search?query=human AND technology_type:"Shotgun proteomics"</a></li>
</ul>



<h3 id="v1-combined-filters">Combined filters</h3>

<p>Any filters can be combined to narrow down the query using the <code>AND</code> operator. More logical operators will be supported in the future.</p>

<h4>Examples</h4>

<ul>
<li><a href='http://localhost:9091/dataset/search?query=human AND technology_type:"Mass Spectrometry" AND publication_date:"2015"'>http://localhost:9091/dataset/search?query=human AND technology_type:"Mass Spectrometry" AND publication_date:"2015"</a></li>
<li><a href='http://localhost:9091/dataset/search?query=human AND technology_type:"Shotgun proteomics" AND modification:"monohydroxylated residue"'>http://localhost:9091/dataset/search?query=human AND technology_type:"Shotgun proteomics" and AND modification:"monohydroxylated residue"</a></li>
</ul>

</div>

  
  <nav id="affix-nav" class="sidebar col-md-3 hidden-print visible-lg" role="complementary">
    <ul style="width: 272px;" class="nav sidenav affix-top" data-spy="affix" id="affix-ul">
        <li>
          <a href="" ng-click="scrollTo('overview')">API Overview</a>
          <ul class="nav">
            <li><a href="" ng-click="scrollTo('web-browsable-api')">Web browsable API</a></li>
            <li><a href="" ng-click="scrollTo('versioning')">Versioning</a></li>
            <li><a href="" ng-click="scrollTo('throttling')">Throttling</a></li>
          </ul>
        </li>
        <li>
          <a ng-click="scrollTo('v1')">API v1 documentation</a>
          <ul class="nav">
            <li>
              <a href="" ng-click="scrollTo('v1-example-responses')">Example responses</a>
              <ul class="nav hide">
                <li><a href="" ng-click="scrollTo('v1-hyperlinked-vs-flat-responses')">Hyperlinked vs flat responses</a></li>
              </ul>
            </li>
            <li><a href="" ng-click="scrollTo('v1-pagination')">Pagination</a></li>
            <li><a href="" ng-click="scrollTo('v1-sort')">Sort</a></li>
            <li><a href="" ng-click="scrollTo('v1-output-formats')">Output formats</a></li>
            <li>
              <a href="" ng-click="scrollTo('v1-filtering')">Filtering</a>
              <ul class="nav">
                <li><a href="" ng-click="scrollTo('v1-filtering-by-database')">Filtering by database</a></li>
                <li><a href="" ng-click="scrollTo('v1-filtering-by-organism')">Filtering by Organism</a></li>
                <li><a href="" ng-click="scrollTo('v1-filtering-by-tissue')">Filtering by Tissue</a></li>
                <li><a href="" ng-click="scrollTo('v1-filtering-by-disease')">Filtering by Disease</a></li>
                <li><a href="" ng-click="scrollTo('v1-filtering-by-modification')">Filtering by Modification</a></li>
                <li><a href="" ng-click="scrollTo('v1-filtering-by-instruments-platform')">Filtering by Instruments & Platforms</a></li>
                <li><a href="" ng-click="scrollTo('v1-filtering-by-publicationdate')">Filtering by Publication Date</a></li>
                <li><a href="" ng-click="scrollTo('v1-filtering-by-technologytype')">Filtering by Technology Type</a></li>
                <li><a href="" ng-click="scrollTo('v1-combined-filters')">Combined filters</a></li>
              </ul>
            </li>
          </ul>
        </li>
    </ul>
  </nav>

</div>





<footer>
    <div id="footer"></div>
</footer>
        
    
 <script src="js/vendor/angular/angular.min.js"></script>
 <script src="js/vendor/angular/angular-animate.min.js"></script>
 <script src="js/vendor/angular/loading-bar/loading-bar.js"></script>
 <script src="js/vendor/underscore/underscore-min.js"></script>
 <script src="js/search/ddi.metadata.search.js"></script>
 <script src="js/vendor/d3/d3.min.js"></script>
 <script src="js/d3/d3.ExpertDatabaseSequenceDistribution.js"></script>
 <script src="js/d3/d3.speciesSunburst.js"></script>
 <script src="js/d3/d3.speciesTree.js"></script>

</body></html>
